app/models/order_item.rb

```diff
@@ -0,0 +1,6 @@
+class OrderItem < ApplicationRecord
+  validates_presence_of :quantity, :price, :fulfilled
+
```

Comment:
We're also able to add validations for numericality for the integers and floats, I prefer to double-validate when possible.

Reply:
I agree, over-validating makes the model more solid.